code/style.css

```diff
@@ -1,7 +1,183 @@
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+	font-family: 'Poppins', sans-serif;
+	height: 100%;
+	margin: 0;
+	text-align: center;
+  width: 100%;
+  color: #ffffff;
+  display: flex;
+  flex-direction: column;
+  background: #d0c9c9;
+}
+
+h1 {
+  font-size: 32px;
+}
+
+h3 {
+ padding: 10px;
+}
+/**Header**/
+
+.hero {
+    background-image: url("pottery.jpg");
+    background-position: center top;
+    background-size: cover;
+    padding: 20% 0;
+    margin-bottom: 30px;
+    font-family: 'Trebuchet MS', 'Helvetica';
+}
+
+/** main **/
+
+video {
+  display: flex;
+  width: 100%;
+}
+
+/** Signup form **/
+
+.signup-form {
+  background: #395d75;
+  border-radius: 2px;
+  margin: 15px;
+}
+
+.signup-form input {
+  background-color: #708a9c;
+  margin: 10px;
+  border-radius: 2px;
+  flex-direction: row;
+}
+
+.signup-form input:valid {
+  /*Could be green for example*/
+}
+
+.signup-form input:invalid {
+  border: solid rgba(252, 53, 53, 0.562);
+}
+
+.info {
+  display: flex;
+  flex-direction: column;
+}
+
+.container {
+  display: inline-grid;
+}
+
+a {
+  color: white;
+}
+
+button {
+  padding: 10px;
+  border-radius: 3px;
+  margin: 10px;
+  border: solid white;
+  background: #395d75;
+  color: aliceblue;
+}
+
+.btn-dark {
+  background-color: rgba(82, 82, 180, 0.856);
+}
+
+
+/**accordion**/
+.accordion {
+  background: rgb(245, 244, 242);
+  color:rgb(105, 105, 151);
+  border-radius: 2px;
+  margin: 15px;
+}
+
+/* active, .question:hover {
+  background-color: rgba(82, 100, 151, 0.404); 
+} */
+
+.accordion div {
+  background:#c8d2d8;
+  color: #395d75;
+  cursor: pointer;
+  padding: 20px;
+  border: solid 1px #395d7536;
+}
+
+.accordion h1 {
+  color: #395d75;
+}
+
+.answer {
+  display: none;
+}
+
+.answer p {
+  margin: 0;
+  padding: 20px;
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
 .active + .answer {
   /* some styling here */
+  display: block;
+  margin: 0;
+  padding: 20px;
+}
+
+@media (min-width: 650px) and (max-width: 1024px) {
```

Comment:
The code that inside of `@media`s opening and closing tag should be indented one step, so it's easy to see what does belong to it